pkg/internal/tools/doc-gen-helper/main.go

```diff
@@ -0,0 +1,104 @@
+//aaago:build exclude
```

Comment:
shouldn't this be //go:build exclude 😅 ?

Reply:
should, the syntax is not highlighted correctly then, so for the time-being I changed it, forgot to remove it :D

Reply:
Removed